src/Entity/Traits/PaymentMethodTrait.php

```diff
@@ -11,9 +11,7 @@
 
 trait PaymentMethodTrait
 {
-    /**
-     * @ORM\OneToMany(targetEntity=Card::class, mappedBy="paymentMethod", orphanRemoval=true)
-     */
+    /** @ORM\OneToMany(targetEntity=Card::class, mappedBy="paymentMethod", orphanRemoval=true) */
     #[ORM\OneToMany(targetEntity: Card::class, mappedBy: 'paymentMethod', orphanRemoval: true)]
```

Comment:
To keep in mind, but Card may should be a supermappedclass and here we should reference an interface. Let's do that on another PR